src/helpers/shared.ts

```diff
@@ -19,3 +19,27 @@ export const shouldSkip = (): { skip: boolean; reason: string } => {
 };
 
 export const wait = (ms: number) => new Promise((r) => setTimeout(r, ms));
+
+export const calculateWeight = (label: LabelItem | undefined): number => {
+  if (!label) return 0;
+  const matches = label.name.match(/\d+/);
+  const number = matches && matches.length > 0 ? parseInt(matches[0]) || 0 : 0;
+  if (label.name.toLowerCase().includes("priority")) return number;
+  if (label.name.toLowerCase().includes("hour")) return number * 0.125;
+  if (label.name.toLowerCase().includes("day")) return 1 + (number - 1) * 0.25;
+  if (label.name.toLowerCase().includes("week")) return number + 1;
+  if (label.name.toLowerCase().includes("month")) return 5 + (number - 1) * 8;
+  return 0;
+};
+
+export const calculateDuration = (label: LabelItem): number => {
```

Comment:
duration calculation will be implemented in #638 so I'm not sure why did you also implement it

Reply:
Original spec is to remove both weight and value.